debian/homestead.init.d

```diff
@@ -122,8 +122,11 @@ get_settings()
         if [ -n "$scscf_uri" ]
         then
           server_name=$scscf_uri
+        elif [ -n "$scscf_prefix" ]
+        then
+          server_name="sip:$scscf_prefix.$sprout_hostname:$scscf;transport=TCP"
         else
-          server_name="sip:$(python /usr/share/clearwater/bin/bracket_ipv6_address.py $sprout_hostname):$scscf;transport=TCP"
```

Comment:
It's not that bug - it's just when sprout_hostname isn't a hostname. It should still 'just work'